sequencer/src/catchup.rs

```diff
@@ -59,15 +63,21 @@ pub trait StateCatchup: Send + Sync + std::fmt::Debug {
         let mut ret = vec![];
         for account in accounts {
             // Retry until we succeed.
+            let mut delay = MIN_RETRY_DELAY;
             let account = loop {
                 match self
                     .try_fetch_account(height, view, fee_merkle_tree_root, account)
                     .await
                 {
                     Ok(account) => break account,
                     Err(err) => {
-                        tracing::warn!(%account, "Could not fetch account, retrying: {err:#}");
-                        async_std::task::sleep(self.retry_interval()).await;
+                        tracing::warn!(%account, ?delay, "Could not fetch account, retrying: {err:#}");
+                        sleep(delay).await;
+
+                        // Try a few times with a short delay, on the off chance that the problem
+                        // resolves quickly. Back off until we eventually reach the maximum delay,
+                        // which should be pretty long.
+                        delay = min(delay * BACKOFF_FACTOR, MAX_RETRY_DELAY);
```

Comment:
Should we also add some jitter?

Reply:
Or maybe the fetch call can take different time each time it is called

Reply:
sure

Reply:
[071e234](https://github.com/EspressoSystems/espresso-sequencer/pull/1525/commits/071e234ec7f0ce1a4e16b3307e428432dea51bdb)